pkg/cli/cli.go

```diff
@@ -44,9 +44,12 @@ func Run(configPath string, inputPath string, outputPath string) error {
 
 	var omitters []omitter.Omitter
 	for _, o := range config.Config.Omit {
-		switch om := o.(type) {
-		case schema.FileOmission:
-			omitters = append(omitters, omitter.NewFilenamePatternOmitter(om.Pattern))
+		switch o.Type {
+		case schema.OmitTypeFile:
+			if *o.Pattern == "" {
```

Comment:
nit: I thought we wanted to add the validations into the NewXX funcs

Reply:
Fixed in [follow-up](https://github.com/openshift/must-gather-clean/pull/21)